components/experimental/src/dimension/provider/count.rs

```diff
@@ -2,30 +2,22 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+/// Represents the plural rules count.
```

Comment:
Can you use `icu::plurals::PluralCategory` instead?

Reply:
done

Reply:
done

Reply:
this can be deleted now?

Reply:
https://github.com/unicode-org/icu4x/pull/5400

Reply:
Please can you just do this in the same PR? I don't understand why you would split this, this enum becomes dead code here.

Reply:
still used here: `components/experimental/src/dimension/provider/currency_compact.rs`

Reply:
then revert this file? these changes don't seem relevant to this PR anymore and it's confusing because there are multiple PRs doing almost the same thing